src/config/media/pipewire.md

```diff
@@ -0,0 +1,66 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+[pipewire(1)](https://man.voidlinux.org/pipewire.1) should be started as a user.
+Use the autostarting mechanism of your desktop environment or
+[startx](../graphical-session/xorg.md#startx).
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+[pipewire.conf(5)](https://man.voidlinux.org/pipewire.conf.5) for auto-starting
+the PulseAudio server, it's recommended to start it from the same place where
+you start PipeWire by executing `pipewire-pulse`. This keeps PipeWire's
+configuration file unmodified for smoother future upgrades.
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and make sure no other
+PulseAudio server instance is running.
```

Comment:
I think the word "After" is not exactly right here, people should check if the pulseaudio server is not running **before** "setting up" pipewire-pulse.

Also, "setting up" is not exactly right, I think it should be something like "starting" or "running".

Reply:
Sounds good